mesonbuild/modules/pkgconfig.py

```diff
@@ -79,14 +79,17 @@ def _process_libs(self, libs, public):
                     if not hasattr(obj, 'generated_pc'):
                         obj.generated_pc = self.name
                     self.add_priv_libs(obj.get_dependencies())
-                    self.add_priv_libs(obj.get_external_deps())
+                    self.add_priv_libs(self.strip_unfound(obj.get_external_deps()))
```

Comment:
Shouldn't we check in dependencies.Dependency and dependencies.PkgConfigDependency cases above too?

Reply:
Updated.